### PR TITLE
Convert to using the CI built data download

### PR DESCRIPTION
**Pull request by hamishcoleman**

### CODE REVIEW

File: hackman_payments/management/commands/import_payments.py [hunk 1]
```diff
@@ -13,16 +12,9 @@ class Command(BaseCommand):
 
     def handle(self, *args, **kwargs):
         p = subprocess.run([
-            'git', '-C', 'dsl-accounts/', 'pull', '-f', 'origin', 'master'
+            'curl', 'curl https://dimsumlabs.github.io/dsl-accounts-pages/paym'
```
Comment (adisbladis) [inline hackman_payments/management/commands/import_payments.py]: We need to save this file to disk so the system can start up properly even without an internet connection.

Comment (hamishcoleman) [reply hackman_payments/management/commands/import_payments.py]: Good point, try this next patch then

**Comment by tomtobback** [comment]
Hi guys, I just saw this pass by, I'm at the door, the WiFi is working, with internet access, the page is telling me 'opened door' but no luck, it's mechanically locked I think

**Comment by adisbladis** [comment]
@tomtobback I'll have a look of that problem today at lunch time.

**Pull request merged**